application/org.openjdk.jmc.flightrecorder.flamegraph/plugin.properties

```diff
@@ -1,6 +1,6 @@
 #
-#  Copyright (c) 2019, Oracle and/or its affiliates. All rights reserved.
-#  Copyright (c) 2019, Datadog, Inc. All rights reserved.
```

Comment:
Since you merely move the files from one project to another, wouldn't it be more appropriate to use copyright `...2019, 2023, ...` here?

Reply:
Actually I didn't move them. Those are new files, but git must detect a rename somehow since I deleted the old plugin in https://github.com/openjdk/jmc/pull/408/commits/b94e708c9abc3f3179bdf013d6f88e6fd592d370.
I can add a rage, but this feels overzealous.

Reply:
Sorry for the typo 😅

Reply:
OK, thanks for the explanation. I agree, copyrights are correct then.